src/bot.ts

```diff
@@ -414,18 +416,49 @@ export class DiscordBot {
         }
         try {
             this.channelLock.set(chan.id);
-            if (!botUser) {
-                // NOTE: Don't send replies to discord if we are a puppet.
+            if (!roomLookup.canSendEmbeds) {
+                // NOTE: Don't send replies to discord if we are a puppet user.
```

Comment:
I've been bitten by this in #522 and we should fix it.

Reply:
still, bot puppets can send embeds just fine, whichlook more nicely. So `!roomLookup.canSendEmbeds` would have a different fallback